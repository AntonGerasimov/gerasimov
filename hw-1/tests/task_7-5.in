2049.00000000093132257461548
