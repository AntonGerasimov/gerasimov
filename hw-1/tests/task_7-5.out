01000000000010101000000000010000000000000000000000000000010000000000000001000000
